Parser/Device/AbstractDeviceParser.php

```diff
@@ -2268,7 +2268,7 @@ protected function parseClientHints(): ?array
                     $deviceType = $formFactors[$formFactor] ?? null;
 
                     if (null !== $deviceType) {
-                        $deviceType = self::getDeviceName($deviceType);
+                        $deviceType = self::getDeviceName($formFactor);
 
                         break;
                     }
```

Comment:
```php

                foreach (self::$clientHintFormFactorsMapping as $formFactor => $deviceType) {
                    $hasDeviceType = $formFactors[$formFactor] ?? null;

                    if (null !== $hasDeviceType) {
                        $deviceType = self::getDeviceName($deviceType);

                        break;
                    }
                }
 ```
                
props clientHintFormFactorsMapping  Contains the order of the device types it must be followed